### DISCUSSION TITLE
vcpkg requirements

### DISCUSSION THREAD

**Discussion by pfuchs** [spinicist/riesling#32] [General]
It could maybe be noted that vcpkg not only requires curl but also zip / unzip (and perhaps build-essentials and pkg-config), since missing these dependencies results in some very confusing errors ( " CMAKE_MAKE_PROGRAM is not set. " and  " CMAKE_CXX_COMPILER not set, after EnableLanguage "). Now typically these packages would be available on a linux but they are not shipped with a default Ubuntu Windows Subsystem for Linux (WSL) install.

I hope this helps someone facing the same confusing error messages as I have.

**Comment by spinicist** [comment]
Hello,

thanks for raising this. I've added those to the README with https://github.com/spinicist/riesling/commit/9922209a6f7041d9e77eeaa1a04377928a789e15

Turns out they are documented for vcpkg here: https://github.com/microsoft/vcpkg#installing-linux-developer-tools
